pkg/server/node.go

```diff
@@ -1468,36 +1473,40 @@ func (s *lockedMuxStream) Send(e *kvpb.MuxRangeFeedEvent) error {
 	return s.wrapped.Send(e)
 }
 
-// MuxRangeFeed implements the kvpb.InternalServer interface.
+// MuxRangeFeed implements the roachpb.InternalServer interface.
 func (n *Node) MuxRangeFeed(stream kvpb.Internal_MuxRangeFeedServer) error {
-	ctx, cancelFeeds := n.stopper.WithCancelOnQuiesce(stream.Context())
-	defer cancelFeeds()
-	rfGrp := ctxgroup.WithContext(ctx)
-
 	muxStream := &lockedMuxStream{wrapped: stream}
+	var streams syncutil.IntMap
```

Comment:
Doesn't seem like this is actually used for anything?

Reply:
You are right; not anymore; though I suspect at some point, some sort of observability around mux spans would be nice. Added a todo for that.